sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1885,6 +1820,80 @@ def _mapping_event_from_change_event(self, status, port_dict):
         helper_logger.log_debug("mapping from {} {} to {}".format(status, port_dict, event))
         return event
 
+    # Update port sfp info and dom threshold in db during xcvrd bootup
+    def _post_port_sfp_info_dom_thr_to_db_during_xcvrd_boootup(self, port_mapping, xcvr_table_helper, stop_event=threading.Event()):
+        # Connect to STATE_DB and create transceiver dom/sfp info tables
+        transceiver_dict = {}
+        retry_eeprom_set = set()
+
+        warmstart = swsscommon.WarmStart()
+        warmstart.initialize("xcvrd", "pmon")
+        warmstart.checkWarmStart("xcvrd", "pmon", False)
+        is_warm_start = warmstart.isWarmStart()
+
+        # Post all the current interface sfp/dom threshold info to STATE_DB
+        logical_port_list = port_mapping.logical_port_list
+        for logical_port_name in logical_port_list:
+            if stop_event.is_set():
+                break
+
+            # Get the asic to which this port belongs
+            asic_index = port_mapping.get_asic_id_for_logical_port(logical_port_name)
+            if asic_index is None:
+                helper_logger.log_warning("Got invalid asic index for {}, ignored while posting SFP info during boot-up".format(logical_port_name))
+                continue
+            rc = post_port_sfp_info_to_db(logical_port_name, port_mapping, xcvr_table_helper.get_intf_tbl(asic_index), transceiver_dict, stop_event)
+            if rc != SFP_EEPROM_NOT_READY:
+                post_port_dom_threshold_info_to_db(logical_port_name, port_mapping, xcvr_table_helper.get_dom_threshold_tbl(asic_index), stop_event)
+
+                # Do not notify media settings during warm reboot to avoid dataplane traffic impact
+                if is_warm_start == False:
+                    notify_media_setting(logical_port_name, transceiver_dict, xcvr_table_helper.get_app_port_tbl(asic_index), port_mapping)
```

Comment:
does cmis manager depend on notify_media_setting? Before the change, notify_media_setting is always called before starting cmis manager; after the change, notify_media_setting may start in parallel of cmis manager.

Reply:
No, CMIS manager does not depent on notify_media_setting.

Reply:
No, it doesn't seem that CMIS manager depends on notify_media_setting.